.github/workflows/pdf.yml

```diff
@@ -78,3 +78,6 @@ jobs:
       with:
         name: PDF_LATEX
         path: docs/_build/latex/book.pdf
+    - uses: actions/checkout@v2
```

Comment:
Checkout is for setting up the `repository` so what you're trying to do here won't work. I would delete this addition and keep the `artifact` upload above. 

The issue here is that the docs are probably built through `readthedocs` so we will need to check how execution is supported there and if `xelatex` is available for building.

Reply:
We should see if `artifacts` are linkable perhaps -- otherwise we may need to save that `artifact` through the `github action` to some linkable location.

Reply:
It looks like the `link` are dependent on `run` numbers so are probably `dynamic`

https://github.com/executablebooks/jupyter-book/suites/2074361830/artifacts/41968814

This [issue](https://github.com/actions/upload-artifact/issues/50) kind of helps but hard to add to the `myst` file. But perhaps we should research a way to get access to these artifacts generated on the `master` branch to get a stable `html link`

Reply:
@choldgraf would you be able to provide us with access to `rtd`?

Reply:
@mmcky I was just trying to initiate a discussion on this with that commit. But I reckon we can also do it in a new PR (after this is merged), to prevent this one from having too many lateral features?

Reply:
sure @AakashGfude we can add this to the docs in a new PR if you like. Then we can freeze this for final review and work out the `rtd` context separately.